SwiftSVG.podspec

```diff
@@ -14,5 +14,5 @@ Pod::Spec.new do |s|
   s.ios.deployment_target = '8.0'
   s.source       = { :git => "https://github.com/mchoe/SwiftSVG.git", :tag => "v#{s.version}" }
   s.source_files  = "SwiftSVG", "SwiftSVG/**/*.{h,swift}"
-
+  s.swift_version = "4.0"
```

Comment:
Would be nice to remove this commit because you add 4.2 in the next commit, just to keep this clean.

Reply:
@DmitryPR Yes, it may be nice. But then I need to create another PR isn't it?

Maybe you could simply use Squash Merge commit? Then you will have 1 commit `Swift 4.2 conversion`.